db/lock.go

```diff
@@ -21,20 +20,20 @@ type Lock struct {
 	LockedAt time.Time `bson:"locked_at"`
 }
 
-// WaitTillAcquireGlobalLock "spins" on acquiring the given database lock,
+// WaitTillAcquireLock "spins" on acquiring the given database lock,
 // for the process id, until timeoutMS. Returns whether or not the lock was
```

Comment:
`timeoutMS` isn't relevant anymore. We just use the `lockTimeout` value

Reply:
fixed.